src/decode-vxlan.c

```diff
@@ -182,6 +182,7 @@ int DecodeVXLAN(ThreadVars *tv, DecodeThreadVars *dtv, Packet *p,
                 tv, dtv, p, pkt + VXLAN_HEADER_LEN, len - VXLAN_HEADER_LEN, DECODE_TUNNEL_VXLAN);
         if (tp != NULL) {
             PKT_SET_SRC(tp, PKT_SRC_DECODER_VXLAN);
+            AFPReadCopyBypass(tp, p);
```

Comment:
I would like to avoid having capture method specific code spill over to non-capture specific code like here

Reply:
done otherwise, but the question is why is the flow bypass callback a member of the packet structure ? (and not the livedev one)

Reply:
I asked the question here : https://github.com/OISF/suricata/pull/13088#discussion_r2059080227